src/Authentication/Authentication.Core/Interfaces/IGraphOptions.cs

```diff
@@ -2,14 +2,11 @@
 //  Copyright (c) Microsoft Corporation.  All Rights Reserved.  Licensed under the MIT License.  See License in the project root for license information.
 // ------------------------------------------------------------------------------
 
-using System;
-using System.Security;
-using System.Security.Cryptography.X509Certificates;
-
 namespace Microsoft.Graph.PowerShell.Authentication
 {
     public interface IGraphOption
     {
         bool EnableWAMForMSGraph { get; set; }
+        bool EnableATPoPForMSGraph { get; set; }
```

Comment:
Design question - have you thought to enable this by default and provide an opt-out at most?

Reply:
Yes, that is the eventual plan. This option would be phased out eventually.